### DISCUSSION TITLE
MMM-SPOTIFY response

### DISCUSSION THREAD

**Discussion by flyinstefan** [skuethe/MMM-Spotify#127] [Q&A] [answered]
Hi,
first of all, since i'm new here, I would like to thank all developers for the effort they put in all those projects. It's amazing what you can do with the community projects.

I figured there is an issue, when you use 
SPOTIFY_SEARCH and the result contains only 1 Object. Then, the item is not played. If Items is > 1, it is played.
Having only 1 item in the result is actually a good thing, since your search was very precise obviously.

LOG:
[17.01.2021 15:31.56.006] [LOG]   magicmirror
[17.01.2021 15:31.56.070] [LOG]   {"type": "album", "query": "strainful train vol 3", "random": false}
[17.01.2021 15:31.56.124] [LOG]   Ich spiele das Album strainful train vol 3.
[17.01.2021 15:31.56.279] [LOG]   200 null {
  albums: {
    href: 'https://api.spotify.com/v1/search?query=strainful+train+vol+3&type=album&offset=0&limit=50',
    items: [ [Object] ],
    limit: 50,
    next: null,
    offset: 0,
    previous: null,
    total: 1
  }
}

Does anyone has an idea what i could do, where to look? I tried, but my skills are definitely not sufficient ;)

EDIT: I just found it, will file an issue.

**Comment by skuethe** [comment, marked as answer]
Was resolved in #128
